subset/network/test_net_min_send

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+REPORT=/tmp/report.txt
+
+FIVE_MINS = 300
+
+ping -n -c 3 $TARGET_IP
+
+python network_min_capture.py $FIVE_MINS
+
+sleep $FIVE_MINS
```

Comment:
I don't understand what this sleep is doing, but it might make more sense if the other file were included.

Reply:
sleep removed, old code, bad code